frontend/src/components/MissionOverview/UpcomingMissionCard.tsx

```diff
@@ -26,13 +27,23 @@ const StyledMissionStartTime = styled.div`
 Icon.add({ more_vertical })
 
 export function UpcomingMissionCard({ mission }: MissionProps) {
+    let navigate = useNavigate()
+    const routeChange = () => {
+        let path = '/mission/' + mission.id
+        navigate(path)
+    }
     return (
-        <StyledMissionCard key={mission.id} variant="default" style={{ boxShadow: tokens.elevation.raised }}>
+        <StyledMissionCard
+            key={mission.id}
+            variant="default"
+            style={{ boxShadow: tokens.elevation.raised }}
+            onClick={routeChange}
+        >
             <HorizontalContent>
                 <Checkbox />
                 <Typography variant="h6">Mission name</Typography>
                 <MissionStartTimeDisplay mission={mission} />
-                <Typography>Tags: 17</Typography>
+                <Typography>Tags: {mission.plannedTasks.length}</Typography>
```

Comment:
The time showed on the cards is wrong.
The time that is sent from backend is in the UTC timezone, so we should either specify this in the card or convert to the local timezone.

See difference in time in screenshots:

Frontend:
![image](https://user-images.githubusercontent.com/57716654/191499312-7747038d-40d5-43d4-acee-4c0665ed5ffa.png)

From Backend (in UTC):
![image](https://user-images.githubusercontent.com/57716654/191499446-78c4abaa-e5db-43f1-8c61-c83f4afe1971.png)

Reply:
Created issue #357